Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md

```diff
@@ -0,0 +1,94 @@
+# Upgrading python interpreter version from 3.7.10 to 3.8.12
+
+Oppia has updated it's python interpreter version 3.7.10 in a phase of switching to a more updated interpreter version 3.8.12. For effective and effortless switching to the newer version, this guide will provide a brief and detailed instruction on how to achieve just that. Please follow the below mentioned instructions carefully.
+
+## Prerequisite
+- This page is dedicated to people who want to upgrade their python interpreter in the existing Oppia directory instead of maintaing a new local repo copy of oppia.
+- Please make sure your local repo as well as remote fork is up-to-date with the develop. 
```

Comment:
Just to check, do they need to merge develop into all their open/active branches (for which they have a PR out), or is this not mandatory?

Reply:
I think it depends. If they are locally working on their PR they would execute the scripts, run test coverage, write lint test etc. In that case the script would throw error since the new installation is done according to python 3.8 , therefore they world need to merge the latest from develop. Otherwise if it doesn't involve testing on local machine, it should not be a compulsion in that case.